backend/apps/readings/analysis.py

```diff
@@ -370,3 +370,114 @@ def compute_median_view_time(self):
             list_of_times.sort()
             median_view_time = statistics.median(list_of_times)
         return round(median_view_time)
+
+    def compute_mean_response_length(self):
+        """
+        Given a list of student response dicts,
+        return the mean character length (across all users) of the response
+        :return: float, mean number of characters in the user's response
+        """
+        list_of_responses = [r.response for r in self.responses]
```

Comment:
There's no advantage to adding this list comprehension -- you can just iterate over `self.responses` and call the `response` param in the loop. And the length of `len(list_of_responses)` == `len(self.responses)` so same deal re: calculating the mean. I'd remove for clarity.